base.fgd

```diff
@@ -1003,6 +1003,8 @@
 	]
 
 	startcolor(color255) : "Starting color" : "0, 0, 0" : "If the map is set to fade in, this determines the color it starts off as."
+	
+	lightmap_queries(boolean) : "Lightmap Queries" : 1 : "If set, lightmap_query_data.kv3 will be generated and included in the VPK for this map. (TODO: Include more details)"

```

Comment:
Let's hold off on this part of the PR until we can better document what this provides for mappers. At the moment all I can see is just a bunch of additional to pack into your map(s).

Reply:
I've made this opt-in by default to resolve this.